docs/options.md

```diff
@@ -101,7 +101,7 @@ The complete set of defaults for the current version of cibuildwheel are shown b
 
 !!! tip
     Static configuration works across all CI systems, and can be used locally if
-    you run `cibuildwheel --plat linux`. This is preferred, but environment
+    you run `cibuildwheel --platform linux`. This is preferred, but environment
```

Comment:
hm. good spot.

Reply:
Technically, unless we turn off argparse's option shortcut feature, this would work. But it's better to use the actual arg name (and [personally not fond](https://github.com/CLIUtils/CLI11#features-not-supported-by-this-library) of that feature).